infrastructure/expected-fail/uncaught-exception.html

```diff
@@ -4,5 +4,6 @@
 <script src="/resources/testharness.js"></script>
 <script src="/resources/testharnessreport.js"></script>
 <script>
+setup({ single_test: true });
```

Comment:
This and the following test was written to test when happens when there's a problem in unwrapped setup code. The expectations in infrastructure/metadata/ are what will have to change with the eventual change to remove implicit single-page tests.

Reply:
Prior to the enforcement of `single_test`, there's no difference between "unwrapped setup code" and "failing single-page test code."

We'll need a new test for the "unwrapped setup code" when we begin enforcing `single_test`. Even then, we'll need to persist this test to preserve coverage for the "failing single-page test code" case.

Reply:
Having tests for both is fine, I'm just saying I want these to keep testing what they were, they're named appropriate for that.

Reply:
The names "uncaught exception" and "unhandled rejection" don't describe the tests' effect on the harness. Until we enforce `single_test`, those events will be interpreted as test failures.

I can add tests for harness errors, but these will be new tests. In order to demonstrate this today, they will have to define a sub-test before producing the uncaught exception or unhandled rejection.

A new commit on this branch introduces those tests.

Reply:
> The names "uncaught exception" and "unhandled rejection" don't describe the tests' effect on the harness.

Indeed, I'm following @zcorpan's style of describing what the test does rather than its pass condition in the naming, since the pass condition may change, and will change here.

Reply:
Can you revert the change to infrastructure/expected-fail/uncaught-exception.html?